google-cloud-examples/src/main/java/com/google/cloud/examples/bigtable/InstanceAdmin.java

```diff
@@ -0,0 +1,217 @@
+    System.out.println("\nGet Instance:");
+    // [START bigtable_get_instance]
+    Instance instance = null;
+    try {
+      instance = adminClient.getInstance(instanceID);
+      System.out.println("Instance ID: " + instance.getId());
+      System.out.println("Instance Meta:");
+      System.out.println("Display Name: " + instance.getDisplayName());
+      System.out.println("Labels:");
+      Map<String, String> labels = instance.getLabels();
+      for (String key : labels.keySet()) {
+        System.out.printf("%s: %s", key, labels.get(key));
+      }
+      System.out.println("State: " + instance.getState());
+      System.out.println("Type: " + instance.getType());
+    } catch (Exception e) {
+      System.out.println("Error getting instance: " + e.getMessage());
+    }
+    // [END bigtable_get_instance]
+    return instance;
+  }
+
+  public static List<Cluster> listClusters(BigtableInstanceAdminClient adminClient,
+      String instanceID) {
+    System.out.println("\nListing Clusters:");
+    // [START bigtable_get_clusters]
+    List<Cluster> clusters = null;
+    try {
+      clusters = adminClient.listClusters(instanceID);
+      for (Cluster cluster : clusters) {
+        System.out.println(cluster.getId());
+      }
+    } catch (Exception e) {
+      System.out.println("Error listing clusters: " + e.getMessage());
+    }
+    // [END bigtable_get_clusters]
+    return clusters;
+  }
+
+  public static void deleteInstance(BigtableInstanceAdminClient adminClient, String instanceID) {
+    System.out.println("\nDeleting Instance:");
+    // [START bigtable_delete_instance]
+    try {
+      adminClient.deleteInstance(instanceID);
+      System.out.println("Instance deleted: " + instanceID);
+    } catch (NotFoundException e) {
+      System.out.println("Error deleting instance: " + instanceID);
+      System.out.println(e.getMessage());
+    }
+    // [END bigtable_delete_instance]
+  }
+
+  public static Cluster addCluster(BigtableInstanceAdminClient adminClient, String instanceID,
+      String clusterID) {
+    Cluster cluster = null;
+    System.out.println("\nAdding cluster to instance: " + instanceID);
+    // [START bigtable_create_cluster]
+    try {
+      cluster = adminClient.createCluster(
+          CreateClusterRequest.of(instanceID, clusterID).setZone("us-central1-c").setServeNodes(3)
+              .setStorageType(StorageType.SSD));
+      System.out.printf("Cluster: %s created successfully", cluster.getId());
+    } catch (Exception e) {
+      System.out.println("Error creating cluster: " + e.getMessage());
+    }
+    // [END bigtable_create_cluster]
+    return cluster;
+  }
+
+  public static void deleteCluster(BigtableInstanceAdminClient adminClient, String instanceID,
+      String clusterID) {
+    // [START bigtable_delete_cluster]
+    System.out.println("\nDeleting Cluster");
+    // [START bigtable_delete_cluster]
```

Comment:
multiple START tag.

Reply:
also do sanity check for number of START and END tag and see it matches and its numbers are same.